rc/tools/spell.kak

```diff
@@ -37,7 +39,7 @@ define-command -params ..1 -docstring %{
         fi
 
         {
-            sed 's/^/^/' "$kak_opt_spell_tmp_file" | eval "aspell --byte-offsets -a $options" 2>&1 | awk '
+            sed 's/^/^/' "$kak_opt_spell_tmp_file" | eval "$kak_opt_spell_cmd $options" 2> /dev/null | awk "$kak_opt_spell_awk_options" '
```

Comment:
I wonder if we should simply ask awk if -b is supported. Then we can probably get rid of the configuration variable.

```sh
awk_options=
awk -b "" >/dev/null 2>&1 && awk_options=-b
```

Only tested with gawk but I'm pretty sure it will not add the -b for BSD awk.

Reply:
Much better, thanks.